### PR TITLE
Baseline glossary entry: mention accessibility and other considerations

### LINKED ISSUE #10350: Consider adding an accessibility disclaimer to the Baseline badge
### Summary

The context for this issue comes from https://github.com/web-platform-dx/web-features/issues/498.

Sorry, this is a long issue to read, but I think the gist of it can be understood by reading these comments:

1. https://github.com/web-platform-dx/web-features/issues/498#issue-2042572563
1. https://github.com/web-platform-dx/web-features/issues/498#issuecomment-1858162816
1. https://github.com/web-platform-dx/web-features/issues/498#issuecomment-1858511106
1. https://github.com/web-platform-dx/web-features/issues/498#issuecomment-1858697605

In short, the [web-features repo](https://github.com/web-platform-dx/web-features) does not currently have a clear guideline around accessibility that's used to decide the Baseline status of a feature. We're discussing ways to change this, but it was suggested in the linked issue above that, in the meantime, it would be good for MDN to add a disclaimer to the Baseline badge.

See https://github.com/web-platform-dx/web-features/issues/498#issuecomment-1858697605 for an example of the proposal. Once expanded, the badge could have something like

```html
 There is no guarantee this feature is
 <a href="https://www.w3.org/TR/WCAG22/#dfn-accessibility-supported">
  accessibility supported
 </a>, however, so support testing is recommended.
```

Because the UI rendering of the Baseline data is owned by MDN, I'm opening this issue for the MDN team to discuss and consider if they want to change the UI and add a note about accessibility.

### URL

N/A

### Reproduction steps

N/A

### Expected behavior

Accessibility should either be included in the baseline status for the feature, or a disclaimer that it's not should be present in the badge.

### Actual behavior

N/A

### Device

Desktop

### Browser

Chrome

### Browser version

Stable

### Operating system

Android

### Screenshot

_No response_

### Anything else?

_No response_

### Validations

- [X] I have read the [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
- [X] I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
- [X] I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/orgs/mdn/discussions/categories/platform).

### PR DESCRIPTION
**Pull request by ddbeck**
### Description

Mention the need for accessibility and other testing in the Baseline _Extra considerations_ section.

### Motivation

There have been a few questions about whether Baseline covers assistive technology. It only covers browser support, not other plugins and tools. This change makes that more explicit.

### Related issues and pull requests

Fixes https://github.com/mdn/yari/issues/10350

### CODE REVIEW

File: files/en-us/glossary/baseline/compatibility/index.md [hunk 1]
```diff
@@ -45,9 +45,16 @@ Baseline tracks availability with the following browsers:
 
 ## Extra considerations
 
-Baseline is a summary guide to support and doesn't cover every situation.
-If your site needs to work with an older device or browser release, or a browser not covered by the Baseline status, then you may need to do your own research or testing.
-If Baseline doesn't cover your situation, then consider reading browser compatibility tables on MDN or visiting [caniuse.com](https://caniuse.com/) before committing to a feature.
+Baseline is a summary guide to browser support, to help you make decisions before committing to a feature.
```
Comment (bsmth) [inline files/en-us/glossary/baseline/compatibility/index.md:48]: Minor reword suggestion for clarity

```suggestion
Baseline is a summary guide to browser support to help you make decisions before committing to use a feature in your projects.
```

File: files/en-us/glossary/baseline/compatibility/index.md [hunk 2]
```diff
@@ -45,9 +45,16 @@ Baseline tracks availability with the following browsers:
...
+But it's not a substitute for testing your application for accessibility, usability, performance, and other considerations.
```
Comment (bsmth) [inline files/en-us/glossary/baseline/compatibility/index.md:49]: Another small edit to consider:

```suggestion
Baseline is not a substitute for testing your application for accessibility, usability, performance, security, or other concerns.
```

File: files/en-us/glossary/baseline/compatibility/index.md [hunk 3]
```diff
@@ -45,9 +45,16 @@ Baseline tracks availability with the following browsers:
...
+
+Baseline may not tell you whether a feature works with:
+
+- Older devices and browser releases
+- Browsers not covered by the Baseline browser set, such as operating system web views
```
Comment (bsmth) [inline files/en-us/glossary/baseline/compatibility/index.md:54]: Can we link to the "Baseline browser set"?

File: files/en-us/glossary/baseline/compatibility/index.md [hunk 4]
```diff
@@ -45,9 +45,16 @@ Baseline tracks availability with the following browsers:
...
+- Assistive technology, such as screen readers
+
+If Baseline doesn't cover your situation, then consider reading browser compatibility tables on MDN and visiting [caniuse.com](https://caniuse.com/) and [a11ysupport.io](https://a11ysupport.io/).
```
Comment (bsmth) [inline files/en-us/glossary/baseline/compatibility/index.md:57]: Copy edit:

```suggestion
If Baseline doesn't cover your situation, you can find more information in the browser compatibility tables on MDN, [caniuse.com](https://caniuse.com/), and [a11ysupport.io](https://a11ysupport.io/).
```

**Review by bsmth — APPROVED** [review]
Thanks Daniel. I'm approving, but there's some comments if you want to consider & incorporate some changes! 👍🏻

File: files/en-us/glossary/baseline/compatibility/index.md (hunk 3 above)
Comment (ddbeck) [reply files/en-us/glossary/baseline/compatibility/index.md:54]: I thought about this, but wasn't sure if it was weird to link to the immediately preceding section of the same page. Happy to do it if that's conventional. Something like this?

```suggestion
- Browsers not covered by the Baseline [browser set](#browser_set), such as operating system web views
```

**Review by ddbeck — COMMENTED** [review]
Thank you, Brian—all good edits. One follow up question for you.

File: files/en-us/glossary/baseline/compatibility/index.md (hunk 3 above)
Comment (bsmth) [reply files/en-us/glossary/baseline/compatibility/index.md:54]: Good point, we're right under the browser set section, so we can omit this 👍🏻

**Pull request merged**